main.go

```diff
@@ -499,8 +498,10 @@ func yaraResults(newwg *sync.WaitGroup) {
 		core.GetSession().Log.Fatal("main: failed to serve: %v", err)
 	}
 	fmt.Println("server inside23 port", *session.Options)
```

Comment:
Can you use `session.Log.` everywhere (this line should be debug level)?
Command line output cannot be parsed otherwise